nodewatcher/modules/frontend/ip_space/templates/network/ip_space/ip_space.html

```diff
@@ -0,0 +1,32 @@
+{% extends "base.html" %}
+
+{% contextblock %}
+    {% load i18n sekizai_tags html_tags url_tags partial_tags leaflet_tags %}
+
+    {% add_data "css_data" "css/ipspace.css" %}
+    {% add_data "js_data" "js/d3.v4.js" %}
+    {% add_data "js_data" "js/point.js" %}
+    {% add_data "js_data" "js/hilbert2d.js" %}
+    {% add_data "js_data_end" "js/jquery.tipsy.js" %}
+    {% add_data "js_data_end" "js/DrawCidrs.js" %}
```

Comment:
Can you rename `DrawCidrs.js` to something like `draw_cidrs.js`? We don't want camel case in filenames.

Reply:
Done